### PR TITLE
ci: print problematic file

### PR DESCRIPTION
**Pull request by SMillerDev**
Last desperate attempt to find where it gets old data from.

### CODE REVIEW

**Review by MikeMcQuaid — APPROVED** [review]
✅ wonder if it's worth adding to `deploy` too

File: .github/workflows/scheduled.yml [hunk 1]
```diff
@@ -70,6 +70,9 @@ jobs:
 
       - name: Archive data
         run: tar czvf data-core.tar.gz _data/ api/ formula/
+        
+      - name: print problem
+        run: cat formula/libidn2.json | jq .bottle.stable
```
Comment (SMillerDev) [inline .github/workflows/scheduled.yml:75]:
```suggestion
        run: cat api/formula/libidn2.json | jq .bottle.stable
```

File: .github/workflows/scheduled.yml [hunk 2]
```diff
@@ -181,6 +184,9 @@ jobs:
 
       - uses: actions/download-artifact@v3
       
+      - name: print problem
+        run: cat formula/libidn2.json | jq .bottle.stable
```
Comment (SMillerDev) [inline .github/workflows/scheduled.yml:188]:
```suggestion
        run: cat api/formula/libidn2.json | jq .bottle.stable
```

File: .github/workflows/scheduled.yml [hunk 3]
```diff
@@ -196,6 +202,9 @@ jobs:
           tar xf data-analytics/data-analytics.tar.gz
           mv api-samples _includes/api-samples
           
+      - name: print problem
+        run: cat formula/libidn2.json | jq .bottle.stable
```
Comment (SMillerDev) [inline .github/workflows/scheduled.yml:206]:
```suggestion
        run: cat api/formula/libidn2.json | jq .bottle.stable
```

File: .github/workflows/scheduled.yml [hunk 4]
```diff
@@ -206,6 +215,9 @@ jobs:
 
       - name: Generate site
         run: bundle exec rake build
+        
+      - name: print problem
+        run: cat formula/libidn2.json | jq .bottle.stable
```
Comment (SMillerDev) [inline .github/workflows/scheduled.yml:220]:
```suggestion
        run: cat api/formula/libidn2.json | jq .bottle.stable
```

**Pull request closed without merging**